plugins/host/src/decl.rs

```diff
@@ -65,20 +65,20 @@ pub struct PluginDeclaration {
 #[macro_export]
 macro_rules! register_plugins {
     ( $indradb_interface_version:expr, $( $name:expr, $t:expr ),* ) => {
-        use indradb_plugin_host::PluginDeclaration;
+        use $crate::PluginDeclaration;
```

Comment:
This line actually isn't necessary, now that I'm looking at it

Reply:
This line is afaik not unnecessary, because the macro is invoked outside the crate, which is the motivation for this PR anyways.